src/plugins/discover/public/application/components/discover_grid/discover_grid_flyout_selection.tsx

```diff
@@ -0,0 +1,99 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import {
+  EuiAccordion,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFlyout,
+  EuiFlyoutBody,
+  EuiFlyoutHeader,
+  EuiIcon,
+  EuiPortal,
+  EuiTitle,
+} from '@elastic/eui';
+import { ElasticSearchHit } from '../../doc_views/doc_views_types';
+import { JsonCodeBlock } from '../json_code_block/json_code_block';
+import { IndexPattern } from '../../../kibana_services';
+
+interface Props {
+  indexPattern: IndexPattern;
+  rows: ElasticSearchHit[];
+  selected: number[];
+  onClose: () => void;
+}
+
+export function DiscoverGridFlyoutSelection({
+  indexPattern,
+  rows,
+  selected,
+
+  onClose,
```

Comment:
this is a draft component to allow the selection of several hits/records ... it's more PoC style, of course it doesn't make sence to show collapsed hits/records with ids, but it's starting point to discuss
<img width="1234" alt="Bildschirmfoto 2020-06-26 um 16 57 28" src="https://user-images.githubusercontent.com/463851/85879176-2d5c5200-b7da-11ea-98b4-55a53b4eef83.png">